.github/workflows/mirror_community_pipeline.yml

```
@@ -86,4 +89,14 @@ jobs:
         run: huggingface-cli upload diffusers/community-pipelines-mirror ./examples/community ${PATH_IN_REPO} --repo-type dataset
         env:
             PATH_IN_REPO: ${{ env.PATH_IN_REPO }}
-            HF_TOKEN: ${{ secrets.HF_TOKEN_MIRROR_COMMUNITY_PIPELINES }}
\ No newline at end of file
+            HF_TOKEN: ${{ secrets.HF_TOKEN_MIRROR_COMMUNITY_PIPELINES }}
+
+      - name: Report success status
+        if: ${{ success() }}
+        run: |
+          pip install requests && python utils/notify_community_pipelines_mirror.py --status=success
```

Haven't tried it myself but `requests` should already be installed since we've installed `huggingface_hub` + uploaded stuff to the Hub using `huggingface_hub` (as it's a success).

I'm fine with reinstalling it though. I understand the point of having it for failures.

```suggestion
          python utils/notify_community_pipelines_mirror.py --status=success
```